ci/buildkite-pipeline.sh

```diff
@@ -227,6 +227,31 @@ EOF
       "downstream-projects skipped as no relevant files were modified"
   fi
 
+  # Downstream Anchor projects backwards compatibility
+  if affects \
+             .rs$ \
+             Cargo.lock$ \
+             Cargo.toml$ \
+             ^ci/rust-version.sh \
+             ^ci/test-stable-perf.sh \
+             ^ci/test-stable.sh \
+             ^ci/test-local-cluster.sh \
+             ^core/build.rs \
+             ^fetch-perf-libs.sh \
+             ^programs/ \
+             ^sdk/ \
+             ^scripts/build-downstream-anchor-projects.sh \
+      ; then
+    cat >> "$output_file" <<"EOF"
+  - command: "scripts/build-downstream-anchor-projects.sh"
+    name: "downstream-anchor-projects"
+    timeout_in_minutes: 30
```

Comment:
This could be shorter until we enable other projects? Seems a shame to wait 30min on something like a connection error.

Reply:
Yeah for sure, shall I try 10 mins and see how it goes?

Reply:
Sgtm!